src/app/tests/TestCommandInteraction.cpp

```diff
@@ -1295,8 +1317,7 @@ void TestCommandInteraction::TestCommandHandlerReleaseWithExchangeClosed(nlTestS
     // Verify that async command handle has been allocated
     NL_TEST_ASSERT(apSuite, asyncCommandHandle.Get() != nullptr);
 
-    // Close the exchange associated with the handle and verify the handle can be gracefully released
-    asyncCommandHandle.Get()->mExchangeCtx->Close();
+    // Close the handle and ensure it's handled correctly.
```

Comment:
But that's not what this test was testing.  This test was testing that if something else closes the exchange (e.g. session release?) the async handle going off the stack doesn't misbehave.  It should keep testing that...  What it's testing after this change is not at all the same thing.

This is a must-fix.

Reply:
Will fix.